styles.css

```diff
@@ -501,3 +501,82 @@ section .portfolio {
 .about-container {
   display: flex;
 }
+
+/* Contact me section */
+
+#contact-section {
+  background: #6070ff;
+  padding-bottom: 7rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 4.6875rem;
+  border-top-right-radius: 7.1875rem;
+  background-image: url(/assets/header-bg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  /* background-position: 20% 40%; */
+}
+
+#contact-section > h2 {
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  display: flex;
+  align-items: center;
+}
+
+#contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: 2rem;
+  width: 447px;
+}
```

Comment:
- [ ] Kindly remove the fixed width, because it brokes your design and makes horizontal scrolling. Make it with percentage.

![image](https://user-images.githubusercontent.com/35535130/165547402-6abd9051-8808-4451-8b2f-b674c4fd84b0.png)

Reply:
Hi there @keroloussamy Thank you very much! I will make that change and submit it again!